client/state/partner-portal/partner/selectors.ts

```diff
@@ -48,7 +48,9 @@ export function getPartnerRequestError( state: PartnerPortalStore ): APIError |
 
 export function isAgencyUser( state: PartnerPortalStore ): boolean {
 	const partner = getCurrentPartner( state );
-	return partner?.partner_type === 'agency';
+	return partner?.partner_type
+		? [ 'agency', 'agency_beta' ].includes( partner.partner_type )
+		: false;
```

Comment:
As we don't have other partner types for now, can we simplify this expression like this:

```
return partner?.partner_type === 'agency' || partner?.partner_type === 'agency_beta'
```